### PR TITLE
[device/celestica]: Add thermalctld support on DX010 platform APIs

### PR DESCRIPTION
**Pull request by mudsut4ke**
**- Why I did it**
- The thermalctld daemon on the Pmon docker requires support from the thermal manager API.

**- How I did it**
- Removed the old function for detecting a faulty fan.
- Removed the old function for detecting excess temperature.
- Implement thermal_manager APIs based on ThermalManagerBase
- Implement thermal_conditions APIs based on ThermalPolicyConditionBase
- Implement thermal_actions APIs based on ThermalPolicyActionBase
- Implement thermal_info APIs based on ThermalPolicyInfoBase
- Add thermal_policy.json

**- How to verify it**
- Check the fan speed during temperature changes.
- Examine events that will occur after the temperature has exceeded the threshold.
- Check for events that will occur after the fan is removed or the fan is not working properly.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [x] 201911
- [x] 202006

### CODE REVIEW

**Comment by mudsut4ke** [comment]
retest this please

**Comment by jleveque** [comment]
Retest vsimage please

File: device/celestica/x86_64-cel_seastone-r0/sonic_platform/thermal_actions.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            {
+                "type": "thermal_control.control"
+                "status": "true"
+            }
+        :param json_obj: A JSON object representing a ControlThermalAlgoAction action.
+        :return:
+        """
+        if ControlThermalAlgoAction.JSON_FIELD_STATUS in json_obj:
+            status_str = json_obj[ControlThermalAlgoAction.JSON_FIELD_STATUS].lower(
+            )
```
Comment (jleveque) [inline device/celestica/x86_64-cel_seastone-r0/sonic_platform/thermal_actions.py:31]: Line break is unnecessary. Suggest moving closing parenthesis to line 30

**Review by jleveque — CHANGES_REQUESTED** [review]

File: device/celestica/x86_64-cel_seastone-r0/sonic_platform/thermal_actions.py (hunk 1 above)
Comment (mudsut4ke) [reply device/celestica/x86_64-cel_seastone-r0/sonic_platform/thermal_actions.py]: updated https://github.com/Azure/sonic-buildimage/pull/6089/commits/63ddb29bf99c6b547bc6c06d892757da2aa88451

**Review by jleveque — APPROVED** [review]

**Pull request merged**